manageiq-operator/deploy/crds/manageiq_v1alpha1_manageiq_crd.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: manageiqs.manageiq.example.com
+spec:
+  group: manageiq.example.com
+  names:
+    kind: Manageiq
```

Comment:
Is it possible to name the `kind` as `ManageIQ`, and subsequently `listKind` to `ManageIQList`?

I'm wondering if this should be the name at all or something else like `manageiq-orchestrator` or something.

@carbonin Thoughts?

Reply:
Also plural `manageiqs` reads funny to me, but I don't know what else it would be.